cmd/ftl/cmd_dev.go

```diff
@@ -360,3 +359,28 @@ func loadGitIgnore(dir string) []string {
 	}
 	return ignore
 }
+
+func compareFileHashes(ctx context.Context, map1, map2 map[string][]byte) bool {
```

Comment:
Might be good to name these "old" and "new"

Reply:
oh good call!